drivers/spi/Kconfig.stm32

```diff
@@ -17,10 +17,18 @@ config SPI_STM32_INTERRUPT
 	help
 	  Enable Interrupt support for the SPI Driver of STM32 family.
 
+config SPI_STM32_DMA
+	bool "STM32 MCU SPI DMA Support"
```

Comment:
do we need this or could it be driven by having dma property in dts for spi nodes?

Reply:
For now, we preferred to keep this way of controlling the feature on top of dts (similarly to SPI_STM32_INTERRUPT). This allows to test both configurations w/o need to change board dts description. I think we'll see in usage if going fully dts driven is more convenient.